elide-standalone/README.md

```diff
@@ -36,23 +37,298 @@ To include `elide-standalone` into your project, add the single dependency:
+ 
+The first models we’ll need are `ArtifactGroup`, `ArtifactProduct`, and `ArtifactVersion`.  For brevity we will omit package names and import statements. 
+
+#### ArtifactGroup.java
+
+  ```java
+  @Include(rootLevel = true, type = "group")
+  @Entity
+  public class ArtifactGroup {
+      @Id
+      public String name = "";
+
+      public String commonName = "";
+
+      public String description = "";
+
+      @OneToMany(mappedBy = "group")
+      public List<ArtifactProduct> products = new ArrayList<>();
+  }
+  ```
+
+#### ArtifactProduct.java
+
+  ```java
+  @Include(type = "product")
+  @Entity
+  public class ArtifactProduct {
+      @Id
+      public String name = "";
+
+      public String commonName = "";
 
-## <a name="usage"></a>Usage
+      public String description = "";
 
-Using Elide standalone out of box is intended to require minimal effort. For persistence, you will minimally need a JPA compatible database (i.e. MySQL), a `Settings` class, and your JPA-annotated data models.
+      @ManyToOne
+      public ArtifactGroup group = null;
 
-### <a name="settings-class"></a>Settings Class
+      @OneToMany(mappedBy = "artifact")
+      public List<ArtifactVersion> versions = new ArrayList<>();
+  }
+  ```
 
-ElideStandalone is configured by implementing the ElideStandaloneSettings interface. Please see the ElideStandaloneSettings class for documentation about fields.
+#### ArtifactVersion.java
 
-Similarly, if you need other metadata across your application, it is important to note that the injector is bound with the following:
+  ```java
+  @Include(type = "version")
+  @Entity
+  public class ArtifactVersion {
+      @Id
+      public String name = "";
 
-```java
-@Inject @Named("elideAllModels") Set<Class> entities;
+      public Date createdAt = new Date();
+
+      @ManyToOne
+      public ArtifactProduct artifact;
+  }
+  ```
+
+### Spin up the API
+
+So now we have some models, but without an API it is not very useful. Before we add the API component, we need to create the schema in the database that our models will use.   Our example uses liquibase to manage the schema.  When Heroku releases the application, our example will execute the [database migrations][https://github.com/aklish/elide-heroku-example/blob/master/src/main/resources/db/changelog/changelog.xml] to configure the database with some test data automatically.  This demo uses Postgres.  Feel free to modify the migration script if you are using a different database provider.
+
+There may be more tables in your database than models in your project.  Similarly, there may be more columns in a table than in a particular model.  Not only will our models work just fine, but we expect that models will normally expose only a subset of the fields present in the database. Elide is an ideal tool for building micro-services - each service in your system can expose only the slice of the database that it requires.
```

Comment:
We can actually also have more models than tables I think.

Reply:
👍